packages/prisma/seed-utils.ts

```diff
@@ -264,3 +265,365 @@ export async function createTeamAndAddUsers(
+        });
+      } else if (attr.type === "SINGLE_SELECT" && attribute.options.length > 0) {
+        const randomOption = attribute.options[Math.floor(Math.random() * attribute.options.length)];
+        await prisma.attributeToUser.create({
+          data: {
+            memberId: member.id,
+            attributeOptionId: randomOption.id,
+          },
+        });
+      } else if (attr.type === "MULTI_SELECT" && attribute.options.length > 0) {
+        // Assign 1-3 random options
+        const numOptions = Math.floor(Math.random() * 3) + 1;
+        const shuffledOptions = [...attribute.options].sort(() => Math.random() - 0.5);
+        const selectedOptions = shuffledOptions.slice(0, numOptions);
+
+        for (const option of selectedOptions) {
+          await prisma.attributeToUser.create({
+            data: {
+              memberId: member.id,
+              attributeOptionId: option.id,
+            },
+          });
+        }
+      }
+    }
+
+    console.log(`\t✅ Assigned ${attr.name} values to ${memberships.length} members`);
+  }
+  return attributeRaw;
+}
+
+export async function seedRoutingForms(
+  teamId: number,
+  userId: number,
+  attributeRaw: { id: string; options: { id: string; value: string }[] }[]
+) {
+  const seededForm = {
+    id: "948ae412-d995-4865-885a-48302588de03",
+    name: `Seeded Form - Insights for ${teamId}`,
+    routes: [
+      {
+        id: "8a898988-89ab-4cde-b012-31823f708642",
+        value: "team/insights-team/team-javascript",
+      },
+      {
+        id: "8b2224b2-89ab-4cde-b012-31823f708642",
+        value: "team/insights-team/team-sales",
+      },
+    ],
+    formFieldFilled: {
+      id: "83316968-45bf-4c9d-b5d4-5368a8d2d2a8",
+    },
+  };
+
+  const form = await prisma.app_RoutingForms_Form.findUnique({
+    where: {
+      id: seededForm.id,
+    },
+  });
+  if (form) {
+    console.log(`Skipping Routing Form - Form Seed, "Seeded Form - Pro" already exists`);
+    return;
+  }
+
+  if (attributeRaw.length === 0) {
+    throw new Error("No attributes found - Please call seedAttributes first");
+  }
+
+  await prisma.app_RoutingForms_Form.create({
+    data: {
+      id: seededForm.id,
+      routes: [
+        {
+          id: seededForm.routes[0].id,
+          action: {
+            type: "eventTypeRedirectUrl",
+            value: seededForm.routes[0].value,
+            eventTypeId: 1133,
+          },
+          queryValue: {
+            id: "aaba9988-cdef-4012-b456-719300f53ef8",
+            type: "group",
+          },
+          attributesQueryValue: {
+            id: "ab99bbb9-89ab-4cde-b012-319300f53ef8",
+            type: "group",
+            children1: {
+              "b98b98a8-0123-4456-b89a-b19300f55277": {
+                type: "rule",
+                properties: {
+                  field: attributeRaw[2].id,
+                  value: [
+                    attributeRaw[2].options.filter((opt) => opt.value === "JavaScript").map((opt) => opt.id),
```

Comment:
This currently relys on the set attributeArray being passed in. Down the line we can make this more flexible of a factory if we really need